app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -174,4 +223,162 @@ private void showUpdateIntervalTimePreferencesDialog() {
         }
         return entries;
     }
+
+
+    public class ViewDialog {
+
+
+        final Context context = getActivity();
+        int string_spinnercase;
+
+        public void showDialog(Activity activity, String msg){
+            final Dialog dialog = new Dialog(activity);
```

Comment:
Please use an AlertDialog. Those have methods to set confirm/cancel buttons which then automatically follow the design of the app.